bitnami/schema-registry/Chart.yaml

```
@@ -34,4 +34,4 @@ maintainers:
 name: schema-registry
 sources:
   - https://github.com/bitnami/charts/tree/main/bitnami/schema-registry
-version: 14.0.1
+version: 14.1.1
```

Thanks for your PR, please note the version was not properly bumped
```suggestion
version: 14.1.0
```